site/content/docs/5.1/components/scrollspy.md

```diff
@@ -168,20 +168,34 @@ Scrollspy also works with nested `.nav`s. If a nested `.nav` is `.active`, its p
 </nav>
 
 <div data-bs-spy="scroll" data-bs-target="#navbar-example3" data-bs-offset="0" tabindex="0">
-  <h4 id="item-1">Item 1</h4>
-  <p>...</p>
-  <h5 id="item-1-1">Item 1-1</h5>
-  <p>...</p>
-  <h5 id="item-1-2">Item 1-2</h5>
-  <p>...</p>
-  <h4 id="item-2">Item 2</h4>
-  <p>...</p>
-  <h4 id="item-3">Item 3</h4>
-  <p>...</p>
-  <h5 id="item-3-1">Item 3-1</h5>
-  <p>...</p>
-  <h5 id="item-3-2">Item 3-2</h5>
-  <p>...</p>
+  <div id="item-1">
+    <h4 id="item-1">Item 1</h4>
```

Comment:
beware of duplicated ids :wink:

Reply:
Oups :confused:  you're right, good catch!. I'll update the PR this morning.

Reply:
Fixed in https://github.com/twbs/bootstrap/pull/36195/commits/7c1b36d8593446b8f76d96ebb667b695bc629638